docs/examples/arrow_example.ipynb

```diff
@@ -195,42 +270,47 @@
    "source": [
     "### Convert the Arrow data to power-grid-model input data\n",
     "\n",
-    "No direct conversion from Arrow Tables to NumPy exists and a copy is always required.\n",
+    "The Arrow record batch or tables can then be converted to row based data or columnar data.\n",
```

Comment:
"The Arrow record batch or tables" sounds weird, but once one realices it is refering to `record_batch` or `tables`, it is more clear. So is it possible to enclose those in `` or something similar to indicate code?